src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -71,14 +71,132 @@ const rpc::PlacementGroupTableData &GcsPlacementGroup::GetPlacementGroupTableDat
 std::string GcsPlacementGroup::DebugString() const {
   std::stringstream stream;
   stream << "placement group id = " << GetPlacementGroupID() << ", name = " << GetName()
-         << ", strategy = " << GetStrategy();
+         << ", strategy = " << GetStrategy()
+         << ", creator job dead: " << placement_group_table_data_.creator_job_dead()
+         << ", creator actor dead: " << placement_group_table_data_.creator_actor_dead();
```

Comment:
`=` or `:` choose one of them, uniform format?

Reply:
Good catch. Will fix!